research-hub-web/src/app/components/cards/document-card/document-card.component.html

```diff
@@ -6,12 +6,8 @@
   <div style="margin-top: 12px"></div>
 
   <mat-card-title>
-    <a
-      class="card-title"
-      [attr.href]="document.url ?? document.document?.url"
```

Comment:
The two url fields are because some documents are hosted on Contentful, and some are just linked to externally, and the respective urls are shown in different fields because of that.